marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -56,29 +57,29 @@ import Ouroboros.Consensus.Shelley.Eras qualified as OEra
 -- * Event
 
 data TxMintEvent = TxMintEvent
-  { txMintEventSlotNo          :: C.SlotNo
-  , txMintEventBlockHeaderHash :: C.Hash C.BlockHeader
-  , txMintEventTxAssets        :: NE.NonEmpty (C.TxId, NE.NonEmpty MintAsset)
+  { txMintEventSlotNo          :: !C.SlotNo
```

Comment:
Do we want these bangs? If we do then there is the `StrictData` language extension that does it for all data constructor fields. But as far as I remember, these help with not building up big thunks in deep loops (like summing a very long list, but not evaluating the sum), but all of our fields are evaluated when the buffer is flushed to database. Also, as the buffer is empty after that then there is no possibility of having a thunk there.

Reply:
Just good practice. I agree they might not be necessary, but there's no harm in using them.